### PR TITLE
 Sequential workflow, User Specific view adaptivecards bot

### PR DESCRIPTION
**Pull request by Madhuri-MSFT**
This sample illustrates sequential workflow, user specific views and upto date adaptive cards bot.

1. Once the app is successfully installed in Teams in a group chat, ping the bot by @mentioning it. Bot will reply with a card showing that the person has initiated the incident. 

  image
  
2. Using refresh activity only the person who has initiated will be able to proceed further by entering the details of the incident and assign it to a person from the group chat, while others in the group chat will still be able to see only the initiated card.

  image

3. User who has initiated the incident will be able to enter the details using the series of cards in a sequential flow and submit it for the further approval/rejection process.

  image
 
4. Once the details are submitted and assigned to a person from the group chat, it will send an updated card to the chat with all the entered details.

  image
  
5. Now, only the person assigned to will be able to either approve or reject it.

  image  

6. After the approval/rejection of the card, the final updated card will be sent to the group chat.

  image

### CODE REVIEW

File: samples/sequential-workflow-adaptive-card/csharp/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+﻿# sequential-workflow-adaptive-card
```
Comment (Trinetra-MSFT) [inline samples/sequential-workflow-adaptive-card/csharp/README.md:1]: App name should start from Uppercase

File: samples/sequential-workflow-adaptive-card/csharp/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+1) Clone the repository
+
+    ```bash
+    git clone https://github.com/OfficeDev/Microsoft-Teams-Samples.git
+    ```
+
+2) If you are using Visual Studio
+  - Launch Visual Studio
+  - File -> Open -> Project/Solution
+  - Navigate to `samples/sequential-workflow-adaptive-card/SequentialUserSpecificFlow` folder
```
Comment (Trinetra-MSFT) [inline samples/sequential-workflow-adaptive-card/csharp/README.md:19]: please rename the folder name to `bot-sequential-flow-adaptive-cards`, update root readMe as well add this sample in Bot section

**Review by Trinetra-MSFT — CHANGES_REQUESTED** [review]

File: samples/sequential-workflow-adaptive-card/csharp/SequentialUserSpecificFlow/Bots/UserSpecificBot.cs [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+            }
+            return null;
+        }
+
+        public AdaptiveCardInvokeResponse createCard(string[] path,InitialSequentialCard data, List<string> subCategory= null)
+        {
+            var cardJson = File.ReadAllText(Path.Combine(path));
+            AdaptiveCardTemplate template = new AdaptiveCardTemplate(cardJson);
+            string subCat1 = "";
+            string subCat2 = "";
```
[the remark this answers is not in the document]
Comment (Madhuri-MSFT) [reply samples/sequential-workflow-adaptive-card/csharp/SequentialUserSpecificFlow/Bots/UserSpecificBot.cs:179]: Yeah that's required to update in payload for adaptive card.

File: samples/sequential-workflow-adaptive-card/csharp/SequentialUserSpecificFlow/Manifest/manifest.json [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    "color": "color.png"
+  },
+  "accentColor": "#60A18E",
+
+  "bots": [
+    {
+      "botId": "<<YOUR-MICROSOFT-APP-ID>>",
+      "needsChannelSelector": false,
+      "isNotificationOnly": false,
+      "scopes": [
```
[the remark this answers is not in the document]
Comment (Madhuri-MSFT) [reply samples/sequential-workflow-adaptive-card/csharp/SequentialUserSpecificFlow/Manifest/manifest.json]: Changed scopes to groupchat and team. Not adding personal scope as it contradicts the user specific flow.

File: samples/sequential-workflow-adaptive-card/csharp/README.md (hunk 2 above)
Comment (Madhuri-MSFT) [reply samples/sequential-workflow-adaptive-card/csharp/README.md]: changed folder name to `bot-sequential-flow-adaptive-cards` . Updated in root readme as well.

**Review by Pothan-MSFT — APPROVED** [review]
Checked and code and looks good.

**Review by Wajeed-msft — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Pull request merged**